packages/web/localizations/es.json

```diff
@@ -910,4 +910,4 @@
     }
   },
   "disclaimer": "Descargo de responsabilidad"
-}
+}
```

Comment:
I think not ending in line breaks will fail the linter IIRC

Reply:
fixed